reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_vehicle_routing_problem_ray_custom.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<blockquote>The city consists of mutually exclusive zones</blockquote><p>The city <strong>is a grid map and</strong> consists of mutually exclusive zones</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>